README.md

```diff
@@ -10,4 +10,6 @@ This repository uses [LFS](https://git-lfs.github.com/) (large file storage) to
 
 *Additional Note:* the GitHub "Download .ZIP" button will not package and deliver the actual binary data of the LFS managed files. You will need to use Git and follow the steps listed above in order to download the input data files.
 
+For more information on downloading and using these files, see the EMOD software documentation at https://institutefordiseasemodeling.github.io/EMOD/.
```

Comment:
This information should be a live link:

```[https://institutefordiseasemodeling.github.io/EMOD/](https://institutefordiseasemodeling.github.io/EMOD/)```

[https://institutefordiseasemodeling.github.io/EMOD/](https://institutefordiseasemodeling.github.io/EMOD/)